### ISSUE TITLE
animations

### ISSUE THREAD

**Issue by xcoderzach** [Amicus/end-dash#15]
animating between states would be coolio.

**Comment by bglusman** [comment]
yeah, could have an data-animation-type on a div or something and make animations a simple plugin system anyone could extend even.
